src/components/holdItem/HoldItem.tsx

```diff
@@ -329,12 +329,10 @@ const HoldItemComponent = ({
       disableMove
         ? 0
         : isActive.value
-        ? withSpring(tY, SPRING_CONFIGURATION)
-        : withTiming(-0.1, { duration: HOLD_ITEM_TRANSFORM_DURATION });
+          ? withSpring(tY, SPRING_CONFIGURATION)
+          : withTiming(-0.1, { duration: HOLD_ITEM_TRANSFORM_DURATION });
 
     return {
-      zIndex: 10,
-      position: 'absolute',
```

Comment:
what does this do?

Reply:
I think because we have these styles already in the `styles.holdItem`